htdocs/admin/agenda_other.php

```diff
@@ -95,17 +95,26 @@
     $commande->thirdparty=$specimenthirdparty;
 
     // Search template files
+	// NB an additional module (mymodule) may be located in :
+	// - dolibarr/htdocs/mymodule
+	// - dolibarr/htdocs/custom/mymodule
+	// so we must check the two locations
     $file=''; $classname=''; $filefound=0;
     $dirmodels=array_merge(array('/'),(array) $conf->modules_parts['models']);
     foreach($dirmodels as $reldir)
     {
-    	$file=dol_buildpath($reldir."core/modules/action/doc/pdf_".$modele.".modules.php",0);
-    	if (file_exists($file))
-    	{
-    		$filefound=1;
-    		$classname = "pdf_".$modele;
-    		break;
-    	}
+		$files = array(dol_buildpath($reldir."core/modules/action/doc/pdf_".$modele.".modules.php",0),dol_buildpath("/custom".$reldir."core/modules/action/pdf_".$modele.".modules.php",0));
```

Comment:
You should not have to add this. Because if your module provide some templates, then $conf->modules_parts['models'] is set and the loop for external module is included into the foreach($dirmodels as $reldir)
You templates must be into
mymodule/core/modules/action/doc/pdf_xxx.modules.php

Whatever is the location of your module is deployed, it should works.
Did you define 'models'=>1 in the descriptor of module ?